index.js

```diff
@@ -313,6 +315,17 @@ function Argv (processArgs, cwd) {
     return options
   }
 
+  var groups = {'Options:': []}
+  self.group = function (opts, groupName) {
+    if (!Array.isArray(opts)) opts = [opts]
+    if (!groups[groupName]) groups[groupName] = []
+    Array.prototype.push.apply(groups[groupName], opts)
```

Comment:
I like this change; for whatever reason I was on a `prototype.apply` binge on the :airplane: ... probably tired.